Sources/RequestsSites.swift

```diff
@@ -36,5 +38,22 @@ public extension APIClient {
 			backoffBehavior: backoffBehavior
 		)
 	}
+    
+    public func fetchSites(_ parameters: [String: String] = [:], backoffBehavior: BackoffBehavior = .wait, completionHandler: @escaping (APIResponse<Site>?, Error?) -> ()) {
+        
+        DispatchQueue(label: "fetchSitesQueue").async {
```

Comment:
There's no need to create a new dispatch queue for every request.  Let's just create one concurrent queue on APIClient and perform asynchronous requests on that.